src/style.css

```diff
@@ -1 +1,77 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src:
+    url('./fonts/Avenir.ttc') format('ttc'),
+    url('./fonts/Avenir-Book.ttf') format('ttf'),
+    url('./fonts/Avenir-Heavy.ttf') format('ttf');
+}
+
+:root {
+  --background-gradient: linear-gradient(to bottom, #fff, #f6f6f6);
+}
+
+html {
+  font-family: Avenir, Arial, sans-serif;
+  color: #3d4e61;
+  font-weight: 400;
+}
+
+body {
+  margin: 0;
+  box-sizing: border-box;
+  padding: 70px 50px 0;
+}
+
+.search-input {
+  width: 100%;
+  color: #3d4e61;
+  border-style: none;
+  box-shadow: 0 1px 8px 0 #3d4e61;
+  outline: none;
+
+  background-image: url('./images/Search.svg');
+  background-repeat: no-repeat;
+}
+
+.search-input:hover {
+  box-shadow: 0 3px 8px 0 #3d4e61;
+}
+
+.search-input:focus {
+  font-weight: 900;
+  background: var(--background-gradient);
+  box-shadow: 0 3px 8px 0 #3d4e61;
+
+  background-image: url('./images/Search.svg');
+  background-size: 19px 19px;
+  background-position: left 25px center;
+  background-repeat: no-repeat;
+}
+
+.field-large {
+  height: 70px;
+  font-size: 16px;
+  line-height: 22px;
+  margin-top: 0;
+  padding: 10px 10px 10px 65px;
+
+  background-size: 19px 19px;
+  background-position: left 25px center;
+}
+
+.field-small {
+  height: 42px;
+  font-size: 14px;
+  line-height: 20px;
+  margin-top: 20px;
```

Comment:
Ensure that the `margin-top` for `.field-small` meets the task requirements for the distance between the search bars. Adjust if necessary.